app/components/windows/EditStreamInfo.vue

```diff
@@ -43,58 +43,38 @@
           :metadata="gameMetadata"
         />
       </h-form-group>
+      <h-form-group v-if="searchProfilesPending">
+        {{ $t('Checking optimized setting for') }} {{ gameModel.value }}...
```

Comment:
this is stale, `gameModel` doesn't have a value anymore, it just is the value.

also, why is this wrapped in a form group?

Reply:
Good catch. It's in `h-form-group` for the alignment under the input above